tests/RawProfilerHook/RawProfilerHook.Tests/TestEngine/TestEngineNoBitness.cs

```diff
@@ -0,0 +1,202 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace RawProfilerHook.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.IO;
+    using System.Text;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using RemoteUnitTestExecutor;
+    using System.Threading;
+
+    /// <summary>
+    ///     Summary description for TestsExtensionHostLoading
+    /// </summary>
+    public class TestEngineNoBitness : RemoteUnitTestExecutor.TestEngineBase
```

Comment:
If I understand correctly, this class is the same as TestEngine, just will slightly different environment variables. Could you fold this back into TestEngine and just have the test configure whether it should use bitness variables or not (maybe as a boolean property on TestEngine, which is read in by OnBeforeStarted)?